runtime/queries/openscad/highlights.scm

```diff
@@ -0,0 +1,60 @@
+(number) @constant.number
+(string) @constant.string
+(boolean) @constant.builtin.boolean
+(include_path) @string
```

Comment:
depending on what this means in the language, `@string.special.path` might be a good option. If it isn't a file-system path then just `@string` is fine